pq-crypto/kyber_r3/verify.c

```diff
@@ -0,0 +1,47 @@
+#include <stddef.h>
+#include <stdint.h>
+#include "verify.h"
+
+/*************************************************
+* Name:        verify
+*
+* Description: Compare two arrays for equality in constant time.
+*
+* Arguments:   const uint8_t *a: pointer to first byte array
+*              const uint8_t *b: pointer to second byte array
+*              size_t len:       length of the byte arrays
+*
+* Returns 0 if the byte arrays are equal, 1 otherwise
+**************************************************/
+int verify(const uint8_t *a, const uint8_t *b, size_t len) {
```

Comment:
We already have the `s2n_constant_time_equals` function. Why not use that?

Reply:
Updated the code to use the s2n functions instead of `verify` and `cmov`.